inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -361,6 +361,26 @@ public function rm( $args, $args_assoc ) {
 		WP_CLI::success( sprintf( 'Successfully deleted %s', $prefix ) );
 	}
 
+	/**
+	 * Ensable the auto-rewriting of media links to S3
+	 */
+	public function enable( $args, $assoc_args ) {
+		if ( false === add_option( 's3_uploads_enabled', 'enabled', '', 'yes' ) ) {
```

Comment:
I strongly doubt that the default will ever change, as this would break backwards compatibility.